src/pages/post-test-summary/post-test-summary.html

```diff
@@ -0,0 +1,62 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>post-test-summary</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content padding class="mes-main-content">
+    <ion-card class="mes-card">
+        <ion-card-header class="mes-main-card-header mes-main-card-header--border-bottom mes-breadcrumbs">
```

Comment:
I'm not sure but you can use either one calls `mes-main-card-header--border-bottom` or `mes-main-card-header`